lib/values.js

```diff
@@ -20,6 +20,8 @@ const ORIGINS /* : Array<string> */ = [ '*' ]
 
 const METHODS /* : Array<string> */ = [ 'DELETE', 'GET', 'OPTIONS', 'PATCH', 'POST', 'PUT' ]
 
+const DEFAULT_TIMEOUT /* : number */ = 15
```

Comment:
is this minutes, seconds, milliseconds? I know because of the change log commit above, but not everyone reads changelogs :) maybe we need a documentation update as well?